src/transformState.js

```diff
@@ -5,7 +5,37 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (let i = 0; i < actions.length; i++) {
+    if (actions[i].type === 'clear') {
+      toClear();
+    }
+
+    if (actions[i].type === 'removeProperties') {
+      toRemove(actions[i].keysToRemove);
+    }
+
+    if (actions[i].type === 'addProperties') {
+      toAdd(actions[i].extraData);
+    }
```

Comment:
You should use the `switch/case` statement as written in checklist
![image](https://user-images.githubusercontent.com/73229127/189703617-732684de-3f92-44c1-835b-a909e8ec649b.png)